src/main/java/com/org/jenkins/custom/jenkins/distribution/service/services/PackagerDownloadService.java

```diff
@@ -21,15 +21,15 @@
     private static Util util = new Util();
 
     /**
-     * @param versionName This is the version name of the war file eg: jenkins-all-latest
      * @param configuration This is the configuration with which the war is generated.
      * @return Response Entity with body as the war file in the form of a resource
      * @throws Exception
      */
-    public ResponseEntity<Resource> downloadWAR(final String versionName, final String configuration) throws IOException, InterruptedException {
+    public ResponseEntity<Resource> downloadWAR(final String configuration) throws IOException, InterruptedException {
         File warFile = null;
         try {
-            warFile = WarGenerator.generateWAR(versionName, configuration);
+            LOGGER.info("We are about to enter the war generator function");
+            warFile = WarGenerator.generateWAR(configuration);
```

Comment:
Is the version removed since at this point "everything" is an unchanging version of a generated war?

Reply:
The version is not removed we are getting it in this manner
THe custom war packager loads the entire config in a `config` variable and what we do is read it like this 
`cfg.getVersionName`